src/app/core/nav-bar/nav-bar.component.html

```diff
@@ -1,40 +1,43 @@
 <div class="icon-bar">
-  <div *ngIf="this.role == 'unregistered'">
-    <a class="active" href="#"><i class="cil-home"></i></a>
-    <a class="active" href="#"><i class="cil-search"></i></a>
-    <a class="active" href="#"><i class="cil-bell"></i></a>
-    <a class="active" href="#"><i class="cil-heart"></i></a>
-    <a class="active" href="#" (click)="$event.preventDefault();onAccountClick()"><i class="at-account"></i></a>
+
+  <div *ngIf="role == 'unregistered'">
+    <a [ngClass]="{'active': activeLink == 'home' }" (click)="onHomeClick()"><i class="cil-home"></i></a>
```

Comment:
It would be better if a router-link was used instead of the a element. There is no need for a click handler then. This applies to the rest of the a elements below.